### PR TITLE
fix pbs.qstat (HPC-6109)

### PR DESCRIPTION
**Pull request by stdweird**
broken in 
https://github.com/hpcugent/vsc-jobs/pull/73/files#diff-85806edb7ed91901e7d7c8818bad7210L92

### CODE REVIEW

**Comment by stdweird** [comment]
retest this please

**Comment by stdweird** [comment]
@JensTimmerman tests_requires via vsc-install works like a charm

File: lib/vsc/jobs/pbs/qstat.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def normalise_exec_host(job, nodes):
     return ",".join(s)
 
 
-def normalise_time(time):
+def normalise_time(job, time): # pylint: disable-msg=W0613
```
Comment (JensTimmerman) [inline lib/vsc/jobs/pbs/qstat.py]: why is it ok to suppres this warning here?
I would be in favour of
`def normalise_time(time, job=None)`

this way you don't need to clean up every call to this function (and your test?)

Comment (JensTimmerman) [inline lib/vsc/jobs/pbs/qstat.py]: ah, I see, you're reverting the previous change that broke this. 
I prefer not suppresing the warning, but put:
`del time  # unused variable`
at the top of the function.
That way the argument is used, but it doesn't pollute the namespace, and is clear to everyone this is the unused argument.

Comment (stdweird) [inline lib/vsc/jobs/pbs/qstat.py]: we need to think of a proper solution in this scenarion, but i don't like your proposals (imagine a new function needs a new argument, then we need to add `del xyz` everywhere). 

maybe define the functions with `**kwargs` magic
```python
def f(**kwargs):
  option_i_use = kwargs.pop('name')
```

and call with all possible named options

Comment (wpoely86) [inline lib/vsc/jobs/pbs/qstat.py]: That makes code way less understandable. I don't see the major issue with the pylint comment? It allows for clean code and it's clear what you mean.

Comment (JensTimmerman) [inline lib/vsc/jobs/pbs/qstat.py]: @wpoely86 it's not clear, I first tought the time argument was unused, but it's the job argument that's unused.

**Pull request merged**